transpose and permute tests fail

```
_______________________________ test_numpy_0090 ________________________________

    def test_numpy_0090():
        x = np.array([[1, 2, 3], [4, 5, 6]])
>       x.transpose(2, 0, 1)
E       ValueError: axes don't match array

tests/test_numpy_0090.py:6: ValueError
______________________________ test_pytorch_0091 _______________________________

    def test_pytorch_0091():
        x = torch.tensor([[1, 2, 3], [4, 5, 6]])
>       x.permute(2, 0, 1); x.transpose(1, 2).tranpose(0, 1)  # 012 -> 021 -> 201
E       RuntimeError: number of dims don't match in permute

tests/test_pytorch_0091.py:6: RuntimeError
```
Seems like the tests assume to be operating on 3D arrays. Got this with numpy 1.17.4 and torch 1.5.0 .

--- a/conversions.yaml
+++ b/conversions.yaml
@@ -123,9 +123,12 @@ shape manipulation:
     pytorch: x.resize_
   - numpy:
     pytorch: x.resize_as_
-  - numpy: x.transpose(2, 0, 1)  # 012 -> 201
+  - numpy: |
+      x = np.arange(6).reshape(3, 2, 1)
+      x.transpose(2, 0, 1)  # 012 -> 201
     pytorch: |
-      x.permute(2, 0, 1); x.transpose(1, 2).tranpose(0, 1)  # 012 -> 021 -> 201
+      x = torch.arange(6).reshape(3, 2, 1)
+      x.permute(2, 0, 1); x.transpose(1, 2).transpose(0, 1)  # 012 -> 021 -> 201
   - numpy: x.flatten
     pytorch: x.view(-1)
   - numpy: x.squeeze()

--- a/run_tests.py
+++ b/run_tests.py
@@ -35,7 +35,7 @@ def main():
     here = osp.dirname(osp.abspath(__file__))
 
     with open(osp.join(here, "conversions.yaml")) as f:
-        data = yaml.load(f)
+        data = yaml.safe_load(f)
 
     for fname in glob.glob(osp.join(here, "tests/*.py")):
         os.remove(fname)

--- a/run_tests.py
+++ b/run_tests.py
@@ -73,7 +73,7 @@ def test_{key}_{id:04d}():
 
     cmd = "pytest -vs tests"
     print("+ %s" % cmd)
-    subprocess.call(cmd, shell=True)
+    subprocess.check_call(cmd, shell=True)
 
 
 if __name__ == "__main__":

